src/layers/sequential.rs

```diff
@@ -33,10 +36,6 @@ where
             .collect()
     }
 
-    pub fn zero_grad(&self) {
-        self.parameters().iter().for_each(|value| value.zero_grad());
-    }
-
```

Comment:
I forgot I removed this when trying to make parameters() return a Vec<f64>, it's unused, but let me know if I should bring it back.

Reply:
This is perfect, optimizers handle this logic now so it should have been removed a while ago, thank you!